src/infinite-hits/infinite-hits.ts

```diff
@@ -12,6 +12,22 @@ import { BaseWidget } from '../base-widget';
 import { NgAisInstantSearch } from '../instantsearch/instantsearch';
 import { noop } from '../utils';
 
+export type InfiniteHitsState = {
+  hits: InfiniteHitsItem[];
+  results: {};
```

Comment:
```suggestion
  results: any;
```

We can't use the `results` object otherwise.

Reply:
very good point, thanks d622c4a